README.md

```diff
@@ -80,6 +82,24 @@ Data lineage is just one type of metadata to help analysis and model training th
 
 <img src="https://lamin-site-assets.s3.amazonaws.com/.lamindb/6sofuDVvTANB0f480001.png" width="850">
 
+You can annotate artifacts with various ontologies. Let’s annotate the artifact with a built-in universal label ontology.
```

Comment:
I think you can drop "built-in" here, because the reader doesn't even know about extendability at this point.

Reply:
done